package/netsurf/package

```diff
@@ -43,6 +45,7 @@ sha256sums=(
 )
 
 build() {
+    ln -s /usr/bin/which /bin
```

Comment:
I'm assuming one of the scripts is trying to run `which` from a specific location that is incorrect?

Reply:
Yes, and `which` was moved from `/bin` to `/usr/bin` in Debian Bullseye (compare https://packages.debian.org/buster/amd64/debianutils/filelist and https://packages.debian.org/bullseye/amd64/debianutils/filelist). This seemed like the fastest way to fix the build to work with our toolchain v2 without delving too much into the specific build system of netsurf.